### PR TITLE
Styling setup

### PR DESCRIPTION
**Pull request by ZeroConfidence**
Styling have been added to the 3 pages quizcreator, quiz runner and quiz stats.
dynamic graph generation has been made for quiz stats.

### CODE REVIEW

File: server/moodle/media/player/videojs/amd/build/.video-lazy.min.js.map.icloud
Comment (DeeKahy) [inline server/moodle/media/player/videojs/amd/build/.video-lazy.min.js.map.icloud:1]: .icloud? what is this?

File: server/moodle/mod/livequiz/quizcreator/Images/trashcan.png
Comment (DeeKahy) [inline server/moodle/mod/livequiz/quizcreator/Images/trashcan.png:1]: epic trash can image, i love it!

File: server/moodledata/temp/theme/boost/all.css
Comment (DeeKahy) [inline server/moodledata/temp/theme/boost/all.css:1]: Don't add this. It should be deleted.

**Comment by DeeKahy** [comment]
oh yeah @ZeroConfidence wrong branch. it should go into staging xd

File: server/moodle/mod/livequiz/quizcreator/Images/trashcan.png
Comment (noobstrikker) [reply server/moodle/mod/livequiz/quizcreator/Images/trashcan.png:1]: det kalder mig MS Paint Master af årsager

**Comment by ZeroConfidence** [comment]
> oh yeah @ZeroConfidence wrong branch. it should go into staging xd

Would you believe that I woke up this morning and immediately thought about that?

**Pull request closed without merging**